src/Bubble.vala

```diff
@@ -174,15 +174,16 @@ public class Notifications.Bubble : AbstractBubble {
             attach (body_label, 1, 1);
 
             if (notification.actions.length > 0) {
-                var action_area = new Gtk.ButtonBox (Gtk.Orientation.HORIZONTAL) {
-                    layout_style = Gtk.ButtonBoxStyle.END
-                };
+                var action_area = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
 
                 bool action_area_packed = false;
 
                 for (int i = 0; i < notification.actions.length; i += 2) {
                     if (notification.actions[i] != "default") {
-                        var button = new Gtk.Button.with_label (notification.actions[i + 1]);
+                        var button = new Gtk.Button.with_label (notification.actions[i + 1]) {
+                            width_request = 85,
+                            height_request = 27
```

Comment:
We probably shouldn't hardcode these values because the UI can scale when text size changes, so this would break that

Reply:
Hi! Should we set it then as minimum through css? I tried without the width and height request property and it looks quite different as it is smaller than buttons inside buttonbox.

Reply:
I pushed a commit setting min-width in CSS. I think probably what should happen is this style class should move to the system stylesheet, so I'll open a branch for that